app/src/main/java/com/kickstarter/ui/activities/UpdateActivity.kt

```diff
@@ -141,7 +141,7 @@ class UpdateActivity : BaseActivity<UpdateViewModel.ViewModel?>(), KSWebView.Del
     }
 
     private fun startCommentsActivity(update: Update) {
-        val intent = Intent(this, CommentsActivity::class.java)
```

Comment:
Well done! 
I'll add the feature flag check on the `UpdatesActivity` as well, I wasn't aware you can also open comments from the "UpdatesActivity" 👍

Reply:
Thanks! Great callout!